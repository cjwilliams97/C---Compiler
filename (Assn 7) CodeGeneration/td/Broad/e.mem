Func: main returns type void [mem: Global loc: 0 size: -2] [line: 1]
.   Child: 1 Compound [mem: None loc: 0 size: -2] [line: 2]
.   .   Child: 1 Call: output of type void [line: 3]
.   .   .   Child: 0 Op: + of type int [line: 3]
.   .   .   .   Child: 0 Const 111 of type int [line: 3]
.   .   .   .   Child: 1 Op: + of type int [line: 3]
.   .   .   .   .   Child: 0 Const 222 of type int [line: 3]
.   .   .   .   .   Child: 1 Op: + of type int [line: 3]
.   .   .   .   .   .   Child: 0 Const 333 of type int [line: 3]
.   .   .   .   .   .   Child: 1 Const 444 of type int [line: 3]
.   .   Sibling: 1 Call: outnl of type void [line: 3]
.   .   Sibling: 2 Call: output of type void [line: 4]
.   .   .   Child: 0 Op: + of type int [line: 4]
.   .   .   .   Child: 0 Op: + of type int [line: 4]
.   .   .   .   .   Child: 0 Op: + of type int [line: 4]
.   .   .   .   .   .   Child: 0 Const 111 of type int [line: 4]
.   .   .   .   .   .   Child: 1 Const 222 of type int [line: 4]
.   .   .   .   .   Child: 1 Const 333 of type int [line: 4]
.   .   .   .   Child: 1 Const 444 of type int [line: 4]
.   .   Sibling: 3 Call: outnl of type void [line: 4]
.   .   Sibling: 4 Call: output of type void [line: 5]
.   .   .   Child: 0 Op: - of type int [line: 5]
.   .   .   .   Child: 0 Const 111 of type int [line: 5]
.   .   .   .   Child: 1 Op: - of type int [line: 5]
.   .   .   .   .   Child: 0 Const 222 of type int [line: 5]
.   .   .   .   .   Child: 1 Op: - of type int [line: 5]
.   .   .   .   .   .   Child: 0 Const 333 of type int [line: 5]
.   .   .   .   .   .   Child: 1 Const 444 of type int [line: 5]
.   .   Sibling: 5 Call: outnl of type void [line: 5]
.   .   Sibling: 6 Call: output of type void [line: 6]
.   .   .   Child: 0 Op: - of type int [line: 6]
.   .   .   .   Child: 0 Op: - of type int [line: 6]
.   .   .   .   .   Child: 0 Op: - of type int [line: 6]
.   .   .   .   .   .   Child: 0 Const 111 of type int [line: 6]
.   .   .   .   .   .   Child: 1 Const 222 of type int [line: 6]
.   .   .   .   .   Child: 1 Const 333 of type int [line: 6]
.   .   .   .   Child: 1 Const 444 of type int [line: 6]
.   .   Sibling: 7 Call: outnl of type void [line: 6]
.   .   Sibling: 8 Call: output of type void [line: 7]
.   .   .   Child: 0 Op: - of type int [line: 7]
.   .   .   .   Child: 0 Op: - of type int [line: 7]
.   .   .   .   .   Child: 0 Op: - of type int [line: 7]
.   .   .   .   .   .   Child: 0 Const 111 of type int [line: 7]
.   .   .   .   .   .   Child: 1 Const 222 of type int [line: 7]
.   .   .   .   .   Child: 1 Const 333 of type int [line: 7]
.   .   .   .   Child: 1 Const 444 of type int [line: 7]
.   .   Sibling: 9 Call: outnl of type void [line: 7]
Offset for end of global space: 0
Number of warnings: 0
Number of errors: 0
