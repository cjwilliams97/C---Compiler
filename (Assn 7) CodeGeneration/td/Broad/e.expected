Loading file: e.tm
1110
1110
-222
-888
-888
Bye.
